superset/views/core.py

```diff
@@ -2014,10 +2014,7 @@ def sync_druid_source(self) -> FlaskResponse:  # pylint: disable=no-self-use
         ]
         DruidCluster = DruidDatasource.cluster_class  # pylint: disable=invalid-name
         if not user:
-            err_msg = __(
-                "Can't find User '%(name)s', please ask your admin " "to create one.",
-                name=user_name,
-            )
+            err_msg = __("Can't find User, please ask your admin to create one.")
```

Comment:
Wondering if that `User` is on purpose or should be `user` instead

Reply:
I agree, `user` is better